libr/core/cmd_anal.c

```diff
@@ -2699,6 +2701,7 @@ static void cmd_esil_mem(RCore *core, const char *input) {
 			if (fd) {
 				cf = r_core_file_get_by_fd (core, fd->offset);
 				r_core_file_close (core, cf);
+				free (cf);
```

Comment:
`cf->map` is not freed. This is intricate. I do not know how to free the whole `cf` properly.